tests/vxingest/builder_common/test_unit_queries.py

```diff
@@ -7,6 +8,7 @@
 from couchbase.auth import PasswordAuthenticator
 from couchbase.cluster import Cluster
 from couchbase.options import ClusterOptions, ClusterTimeoutOptions, QueryOptions
+from vxingest.grib2_to_cb.run_ingest_threads import VXIngest
```

Comment:
I didn't realize this at the time, but we shouldn't be testing `grib2_to_cb` module components in the tests for the `builder_common` module. We should move this to `tests/vxingest/grib2_to_cb` so that the tests stay focused on particular areas of the source code.

Otherwise, I wonder if we can `from vxingest.builder_common.vxingest import CommonVxIngest` and just call `get_file_list()` directly.